src/component/SignUpForm/SignUpForm.jsx

```diff
@@ -1,44 +1,44 @@
 import './SignUpForm.css'
 
-export default function SignUpForm() {
+export function SignUpForm() {
  return (
   <>
     <h1>Crear cuenta</h1>
-   
-    <div className='sign-up-form'>
-      <form>
-        <div>
-          <label htmlFor="email">E-mail</label>
-          <input type="text" id="email" name="email" required/>
-        </div>
-
-        <div>
-          <label htmlFor="name">Nombre</label>
-          <input type="text" id="name" name="name" required/>
-        </div>
-
-        <div>
-          <label htmlFor="password">Contraseña</label>
-          <input type="password" id="password" name="password" required/>
-        </div>
-
-        <div>
-          <label htmlFor="confirm-password">Confirmar contraseña</label>
-          <input type="password" id="confirm-password" name="confirm-password" required/>
-        </div>        
-
-        <div>
-          <label htmlFor="terms-and-conditions">Acepta los términos y condiciones</label>
-          <input type="checkbox" id="terms-and-conditions" name="terms-and-conditions" required/>
-        </div>  
-
-        <div id="button">
-            <button type="submit">Crear cuenta</button> 
-        </div>
-
-      </form>
+    <div className="center-container">
+      <div className='sign-up-form'>
+        <form>
+          <div className='form-group'>
+            <label htmlFor="email">E-mail</label>
+            <input type="email" id="email" name="email" required/>
+          </div>
+
+          <div className='form-group'>
+            <label htmlFor="name">Nombre</label>
+            <input type="text" id="name" name="name" required/>
+          </div>
```

Comment:
Como pueden ver en este componente se van repitiendo la estructura de los inputs con sus labels, su contenedor y la clase css form-group. 
A su vez, me imagino que esta misma estructura la usaran en el form de Sign in. 

Dicho esto podrían abstraer esta logica repetida en un componente generico, asi si precisan modificar algo lo pueden hacer en un solo lugar. El componente generico podría ser algo asi: 
```
export function Input ({  id, type, label, name, required, ...props - si es que necesitan alguna otra prop }) {
  return (
         <div className='form-group'>
            <label htmlFor={id}>{label}</label>
            <input type={type} id={id} name={name} required={required} />
          </div>
  )
}
```
y en el css de ese mismo componente definen la clase `form-group`. 

Es un comentario no más que les puede servir, pero no quiero bloquearles mergear este PR este por ello. Cualquier cosa lo pueden agregar en otro PR!